include/capstone/loongarch.h

```diff
@@ -0,0 +1,239 @@
+#ifndef CAPSTONE_LOONGARCH_H
+#define CAPSTONE_LOONGARCH_H
+
+/* Capstone Disassembly Engine */
+/* By Jiajie Chen <c@jia.je> 2024 */
+/*    Yanglin Xun <1109673069@qq.com>, 2024 */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if !defined(_MSC_VER) || !defined(_KERNEL_MODE)
+#include <stdint.h>
+#endif
+
+#include "platform.h"
+
+//> Operand type for instruction's operands
+typedef enum loongarch_op_type {
+	LOONGARCH_OP_INVALID = 0, // = CS_OP_INVALID (Uninitialized).
+	LOONGARCH_OP_REG, // = CS_OP_REG (Register operand).
+	LOONGARCH_OP_IMM, // = CS_OP_IMM (Immediate operand).
+	LOONGARCH_OP_MEM, // = CS_OP_MEM (Memory operand).
```

Comment:
```suggestion
	LOONGARCH_OP_REG = CS_OP_REG ///< (Register operand).
	LOONGARCH_OP_IMM = CS_OP_IMM ///< (Immediate operand).
	LOONGARCH_OP_MEM = CS_OP_MEM ///< (Memory operand).
```

You need to include
```c
#include "cs_operand.h"
```
above for this.